### PR TITLE
DOC: Hub reporting for beta1

### PR DESCRIPTION
**Pull request by mbussolotto**
# Description

Hub reporting documentation for beta1

# Target branches

Which documentation version does this PR apply to?

- [x] Master (Default)
- [ ] Manager-4.2
- [ ] Manager-4.1
- [ ] Manager-4.0

# Links

Fixes https://github.com/SUSE/spacewalk/issues/16596

### CODE REVIEW

File: modules/specialized-guides/pages/large-deployments/hub-reporting.adoc [hunk 1]
```diff
@@ -0,0 +1,57 @@
+[[lsd-hub-reporting]]
+= Hub Reporting
+The Hub prepares and provides content for multiple peripheral {productname} Servers. The goal is now to get data from these Servers back and have combined reporting data available on the Hub.
+The data are made available for external Reporting Tools.
```
Comment (mcalmer) [inline modules/specialized-guides/pages/large-deployments/hub-reporting.adoc:4]: When I use here  "View file" I cannot see this paragraph.

File: modules/specialized-guides/pages/large-deployments/hub-reporting.adoc [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+== Architecture
+The main database is a PostgresDB in the {productname} Hub system.  It stores all the information collected from all the servers, and eventually aggregates them. 
+Other databases with the same schema are also present in all the servers, to collect information for that system. In summary:
+
+- the DB in {productname} Hub stores, collects and eventually aggregates data coming from all the DBs of the peripheral Servers,
+- the DB in {productname} Hub stores also its own data 
+- data from the systems directly connected and managed by the Hub,
```
Comment (mcalmer) [inline modules/specialized-guides/pages/large-deployments/hub-reporting.adoc:12]:
```suggestion
- the DB in {productname} Hub stores also its own data from the systems directly connected and managed by the Hub,
```

File: modules/specialized-guides/pages/large-deployments/hub-reporting.adoc [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+- the DB in peripheral {productname} Server stores its own data,
+- the reporting tool can be connected either to the Hub or to any {productname} Server.
+
+== Setup
+Database and schema are setup by default.
```
Comment (mcalmer) [inline modules/specialized-guides/pages/large-deployments/hub-reporting.adoc:17]:
```suggestion
The reporting database and schema are setup by default using the local postgresql server.
The reporting database is a separate database accessible via the network.
```

File: modules/specialized-guides/pages/large-deployments/hub-reporting.adoc [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+            Delete the user
+  --modify
+            Set a new password
+----
+
+== Database Schema
+
+The schema export the most important tables from the main {productname} Database as a de-normalized variant containing only data which are relevant for a report.
+
+A ready-to-use report can be provided with views doing joins over multiple tables.
```
Comment (mcalmer) [inline modules/specialized-guides/pages/large-deployments/hub-reporting.adoc:48]:
```suggestion
A ready-to-use reports are provided with views doing joins over multiple tables.
```

File: modules/specialized-guides/pages/large-deployments/hub-reporting.adoc [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+
+Every table gets an extra column for the {productname} server id of server which provide the data `mgm_id`.
```
Comment (mcalmer) [inline modules/specialized-guides/pages/large-deployments/hub-reporting.adoc:50]:
```suggestion
Every table gets an extra id column (`mgm_id`) for the {productname} server which provided the data. 
```

File: modules/specialized-guides/pages/large-deployments/hub-reporting.adoc [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+----
+
+== Database Schema
+
+The schema export the most important tables from the main {productname} Database as a de-normalized variant containing only data which are relevant for a report.
+
+A ready-to-use report can be provided with views doing joins over multiple tables.
+
+Every table gets an extra column for the {productname} server id of server which provide the data `mgm_id`. 
+On a single {productname} Server this column has the standard value 1 which represent "localhost".
```
Comment (mcalmer) [inline modules/specialized-guides/pages/large-deployments/hub-reporting.adoc:51]:
```suggestion
On a single {productname} Server this column has the standard value `1` which represent "localhost". 
```

File: modules/specialized-guides/pages/large-deployments/hub-reporting.adoc [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+            Delete the user
+  --modify
+            Set a new password
+----
+
+== Database Schema
+
+The schema export the most important tables from the main {productname} Database as a de-normalized variant containing only data which are relevant for a report.
+
+A ready-to-use reports are provided with views doing joins over multiple tables.
```
Comment (mackdk) [inline modules/specialized-guides/pages/large-deployments/hub-reporting.adoc:49]:
```suggestion
Ready-to-use reports are provided as views, aggregating data over multiple tables.
```

File: modules/specialized-guides/pages/large-deployments/hub-reporting.adoc [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+
+Every table gets an extra id column (`mgm_id`) for the {productname} server which provided the data.
```
Comment (mackdk) [inline modules/specialized-guides/pages/large-deployments/hub-reporting.adoc:51]:
```suggestion
Every table gets an extra id column (`mgm_id`) specifying the {productname} server which provided the data. 
```

File: modules/specialized-guides/pages/large-deployments/hub-reporting.adoc [hunk 9]
```diff
@@ -0,0 +1,58 @@
...
+            Add the new user
+  --delete
+            Delete the user
+  --modify
+            Set a new password
+----
+
+== Database Schema
+
+The schema export the most important tables from the main {productname} Database as a de-normalized variant containing only data which are relevant for a report.
```
Comment (mackdk) [inline modules/specialized-guides/pages/large-deployments/hub-reporting.adoc:47]:
```suggestion
The schema exports the most important tables from the main {productname} Database as a de-normalized variant containing only data which are relevant for a report.
```

File: modules/specialized-guides/pages/large-deployments/hub-reporting.adoc [hunk 10]
```diff
@@ -0,0 +1,58 @@
...
+
+A ready-to-use reports are provided with views doing joins over multiple tables.
+
+Every table gets an extra id column (`mgm_id`) for the {productname} server which provided the data. 
+On a single {productname} Server this column has the standard value `1` which represent "localhost". 
+On the {productname} Hub it is replaced with the real server id the managed server has in the hub database.
+
+The `synced_date` represent the time when the data were exported from the main {productname} Server database.
```
Comment (mackdk) [inline modules/specialized-guides/pages/large-deployments/hub-reporting.adoc:55]:
```suggestion
Another common additional field is `synced_date`, which represent the time when the data were exported from the main {productname} Server database.
```

File: modules/specialized-guides/pages/large-deployments/hub-reporting.adoc [hunk 11]
```diff
@@ -0,0 +1,58 @@
+[[lsd-hub-reporting]]
+= Hub Reporting
+
+The Hub prepares and provides content for multiple peripheral {productname} Servers. The goal is now to get data from these Servers back and have combined reporting data available on the Hub.
```
Comment (mackdk) [inline modules/specialized-guides/pages/large-deployments/hub-reporting.adoc:4]:
```suggestion
The Hub prepares and provides content for multiple peripheral {productname} Servers. The goal of the reporting feature is to get data from these Servers back and have combined reporting data available on the Hub.
```

File: modules/specialized-guides/pages/large-deployments/hub-reporting.adoc [hunk 12]
```diff
@@ -0,0 +1,58 @@
+[[lsd-hub-reporting]]
+= Hub Reporting
+
+The Hub prepares and provides content for multiple peripheral {productname} Servers. The goal of the reporting feature is to get data from these Servers back and have combined reporting data available on the Hub.
+The data are made available for external Reporting Tools.
```
Comment (jcayouette) [inline modules/specialized-guides/pages/large-deployments/hub-reporting.adoc:5]: The data is

**Review by jcayouette — APPROVED** [review]

**Review by 0rnela — APPROVED** [review]

**Pull request merged**